frontend/components/components.json

```
@@ -13,7 +13,7 @@
   "aliases": {
     "components": "@/components",
     "utils": "@/lib/utils",
-    "ui": "@/base/ui",
+    "ui": "@/base",
```

Pourquoi on garde l'alias ui vu qu'on retire le dossier ?
```suggestion
    "base": "@/base",
```